opencencus-check-listener/utils.go

```diff
@@ -0,0 +1,13 @@
+package opencencus
+
+import gosundheit "github.com/AppsFlyer/go-sundheit"
+
+func allHealthy(results map[string]gosundheit.Result) (healthy bool) {
```

Comment:
I guess we need a cleanup iteration and refactor `map[string]gosundheit.Result` into a type, and add this fucntion there